CMakeLists.txt

```diff
@@ -420,7 +420,18 @@ if (OPM_ENABLE_PYTHON)
                   OUTPUT_VARIABLE python_lib_target)
 
   add_custom_target(copy_python ALL
-    COMMAND ${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/python/install.py ${PROJECT_SOURCE_DIR}/python ${PROJECT_BINARY_DIR} 0)
+      COMMAND ${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/python/install.py
+        ${PROJECT_SOURCE_DIR}/python/opm ${PROJECT_BINARY_DIR}/python 0
+      COMMAND ${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/python/install.py
+        ${PROJECT_SOURCE_DIR}/python/cxx ${PROJECT_BINARY_DIR}/python 0
+      COMMAND ${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/python/install.py
+        ${PROJECT_SOURCE_DIR}/python/tests ${PROJECT_BINARY_DIR}/python 0
+      COMMAND ${Python3_EXECUTABLE} ${PROJECT_SOURCE_DIR}/python/install.py
+        ${PROJECT_SOURCE_DIR}/python/examples ${PROJECT_BINARY_DIR}/python 0
+      # install.py is used by opm-simulators if not building against an installed opm-common
+      COMMAND ${CMAKE_COMMAND} -E copy_if_different
+        ${PROJECT_SOURCE_DIR}/python/install.py ${PROJECT_BINARY_DIR}/python
```

Comment:
I think we can skip copying this, can't we?

Reply:
Currently we need to copy `install.py` to the build directory in the case we use sibling build, then `opm-simulators` uses the cmake variable `opm-common_DIR` which will refer to the build direcory of `opm-common`, see [line 59](https://github.com/OPM/opm-simulators/blob/9f7de03a8073445ddb96c2f9244c1a019fc8ebb9/python/simulators/CMakeLists.txt#L59) in `python/simulators/CMakeLists.txt`.

Reply:
mmh. I see.